hw/top_earlgrey/data/ip/chip_i2c_testplan.hjson

```diff
@@ -0,0 +1,94 @@
+
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_kmac
+  testpoints: [
```

Comment:
Should we add an explicit check for clock stretching to one of the tests?

Reply:
Clock stretching is important, but I also suggest adding a test for NAK detection.

Reply:
Thanks for these suggestions, I tried to avoid specifying the whole I2C protocol in the feature list, but I agree that clock stretching and nack detection are important corner cases to test. So I have added them to the feature list and added two tests for them. Let me know what you think of them.